miso-web/src/main/webapp/scripts/hot_library.js

```diff
@@ -167,7 +161,7 @@ HotTarget.library = (function() {
             data: 'sampleBoxPositionLabel',
             type: 'text',
             readOnly: true,
-            include: config.sortableLocation,
+            include: Constants.isDetailedSample && config.sortableLocation,
```

Comment:
Why is sample location unique to detailed samples?

Reply:
It's on the `DetailedLibraryDto`

Reply:
There doesn't seem to be a reason for that though. The data comes from plain sample fields unless I'm missing something?

Reply:
Please try moving to plain Library as we discussed. We can add an option in miso.properties to show/hide the column in the future if that is requested.

It would be ideal to enable sorting for plain samples as well, but if handsontable makes that difficult with all of the undefined values, it can also wait until/if requested